lib/src/test/java/io/ably/lib/test/realtime/EventEmitterTest.java

```diff
@@ -175,4 +176,30 @@ public void once_off_event_simple() {
 		assertNull(listener.counts.get(MyEvents.event_1));
 	}
 
```

Comment:
There needs to be a comment here please with a reference to the requirement `RTE6a`